CONTRIBUTING.md

```diff
@@ -15,7 +15,12 @@ section.
 * Use of the singular form in module name (or use "multi"),
   except when compound of module name or object Odoo
   that is already in the plural (i.e. mrp_operations_....).
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
+* In the `__openerp__.py`  manifest file:
+  * Avoid empty keys
+  * Make sure it has the `license` key
+  * Make sure the text `,Odoo Community Association (OCA)` is appended
```

Comment:
It works both ways.

In apps.odoo.com, they look the same, as authors are split are redraw, but in Odoo module list, the split is not made, and you see all the author list without spaces. 

Current problem is that we cannot put more than 128 characters on this string, so commas are another character we lose. I put it an issue in https://github.com/odoo/odoo/issues/5850, but it's not treated yet. Maybe you can speed up this.